### ISSUE TITLE
False positive on package name check

### ISSUE THREAD

**Issue by Tapchicoma** [shyiko/ktlint#276]
I have a test module that contains two files.

Ktlint (0.27.0) produces following error for both files:
`Package directive doesn't match file location (cannot be auto-corrected) (package-name)`

while this files has valid package and AS doesn't suggest me to correct it:
```kotlin
package com.freeletics.core.tracking.test
```

Here is a log from ktlint check:
```
[DEBUG] Discovered ruleset "standard"
[DEBUG] Discovered reporter "plain"
[DEBUG] Discovered reporter "json"
[DEBUG] Discovered reporter "checkstyle"
[DEBUG] Initializing "plain" reporter with {verbose=true, color=false}
[DEBUG] Initializing "checkstyle" reporter with {verbose=true, color=false}, output=/work/freeletics-android/test/tracking/build/reports/ktlint/ktlint-debug.xml
[DEBUG] Checking /work/freeletics-android/test/tracking/src/main/java/com.freeletics.core.tracking.test/TestTracker.kt
[DEBUG] Checking /work/freeletics-android/test/tracking/src/main/java/com.freeletics.core.tracking.test/TestTrackerAssertions.kt
[DEBUG] Discovered .editorconfig (/work/freeletics-android) {insert_final_newline=true, max_line_length=120}
/work/freeletics-android/test/tracking/src/main/java/com.freeletics.core.tracking.test/TestTracker.kt:1:1: Package directive doesn't match file location (cannot be auto-corrected) (package-name)
/work/freeletics-android/test/tracking/src/main/java/com.freeletics.core.tracking.test/TestTrackerAssertions.kt:1:1: Package directive doesn't match file location (cannot be auto-corrected) (package-name)
"checkstyle" report written to /work/freeletics-android/test/tracking/build/reports/ktlint/ktlint-debug.xml
[DEBUG] 728ms / 2 file(s) / 2 error(s)
```

**Comment by shyiko** [comment]
Fixed in [0.28.0](https://github.com/shyiko/ktlint/blob/master/CHANGELOG.md#0280---2018-09-05).

**Comment by Tapchicoma** [comment]
Damn, missed that package directory is with dots. Anyway thank you for fix!

**Comment by BulatMukhutdinov** [comment]
I am using 0.28.0 but still get this error. I have shred dir for tests called testShared and one class inside this dir. 
Error is 
`/Users/bulat_mukhutdinov/StudioProjects/asia-chat/asiapay/src/testShared/MockData.kt:1:1: Package directive doesn't match file location (cannot be auto-corrected)
`
